### ISSUE TITLE
Feature Request: Show DLC next to the game

### ISSUE THREAD

**Issue by liamdawe** [desura/desura-app#968]
It seems really odd to have DLC listed as their own item seperated from the main game in the Play list.

It needs to be with the game ideally, or in a pop-up list menu.

Another possible solution is a tree-list, when you have a plus button to expand a game to see DLC.

**Comment by Tokiolabs** [comment]
Agreed, this may just end up being a bigger issue in terms of implementation and would effect a lot of how the backend is setup.  It may have to be a drop-down list of sorts, in case a game has a great deal of DLC associated with it.

All in all, a better way of handling it though.  I think when we get through the issues, spending time with the various UI/UX enhancements will be hot on the list

**Comment by liamdawe** [comment]
Thinking on this, why would it need backend work? You must already have identifiers to class it as DLC for a certain game, so you should only need that to place it in an expandable list for each game?

**Comment by Tokiolabs** [comment]
@liamdawe You give us way too much credit!  It's a matter of how DLC is managed at Desura itself; unfortunately the client doesn't have any information to distinguish it as DLC, since from Desura's standpoint this is just another game title.  We would need to send some sort of flag over (and of course make some UI adjustments).

**Comment by liamdawe** [comment]
I see, yes, I thought you would have a simple "is_dlc" boolean and an app id identifier in your database that is passed to the client to pick up on.

**Comment by Tokiolabs** [comment]
@liamdawe Unfortunately; there's a lot of things there -- but currently that's not one of them.  There's a few points of data throughout the site that would be nice to haves (this of course being one of them).
